### DISCUSSION TITLE
1k stars ✨. Thank you so much for your tremendous support! 🙏🙏🙏

### DISCUSSION THREAD

**Discussion by Tao-VanJS** [vanjs-org/van#26] [Announcements]
Hi Fellow VanJSers,

The tremendous support we received from the great VanJS community has greatly exceeded even our wildest expectations! We hit the milestone of 1k stars within 2 weeks of the initial announcement.

I will try my best to ensure everyone a smooth coding experience with VanJS, and stay committed to our mission: *Enabling everyone to build UI apps with a few lines of code, anywhere, any time, on any device.*

Thank you so much!!! 🙏🙏🙏 

Meanwhile, we're the hottest JavaScript project right now. 🚀🚀🚀

![screemshot](https://vanjs.org/code/bestofjs.png)

**Comment by artydev** [comment]
You diserve it :-)

**Reply by Nakamcode** [reply]
2K very soon 🚀
